Library/Homebrew/cask/artifact/installer.rb

```diff
@@ -18,9 +18,9 @@ class Installer < AbstractArtifact
       module ManualInstaller
         def install_phase(**)
           puts <<~EOS
-            To complete the installation of Cask #{cask}, you must also
-            run the installer at:
-              #{cask.staged_path.join(path)}
+            Cask #{cask} only provides a manual installer, please
+            open #{cask.staged_path.join(path).to_s.shellescape}
+            to run the installer and complete the installation.
```

Comment:
```suggestion
            Cask #{cask} only provides a manual installer.
            To run it and complete the installation please run:
              open #{cask.staged_path.join(path).to_s.shellescape}
```

or something similar that results on just the command to run indented as the last line